packages/core/src/amazonqFeatureDev/session/session.ts

```diff
@@ -84,6 +91,8 @@ export class Session {
             span.record({ amazonqConversationId: this._conversationId, credentialStartUrl: AuthUtil.instance.startUrl })
         })
 
+        this.showDevPortal(msg)
```

Comment:
should this be a static title?

Reply:
No, how users distinguish different task